closed/autoconf/custom-spec.gmk.in

```diff
@@ -89,7 +89,6 @@ COMPILER_VERSION_STRING := @COMPILER_VERSION_STRING@
 
 include $(TOPDIR)/closed/openjdk-tag.gmk
 
-J9JDK_EXT_VERSION       := $(VERSION_NUMBER_FOUR_POSITIONS)
```

Comment:
Having `$(VERSION_NUMBER_FOUR_POSITIONS)` as (effectively) a comment is handy, as this is needed when doing a release. If it's not there as a comment we need to look up what it should be, which is more time consuming.

Reply:
Frankly, I don't see how `$(VERSION_NUMBER_FOUR_POSITIONS)` helps much. The document that explains that this file should change for a release should make it clear what value to give `J9JDK_EXT_VERSION` (perhaps by consulting `openjdk-tag.gmk`).

I think we need to do something - this change is already in jdknext. I never understood the desire for development and nightly build to say `HEAD` here: such builds are clearly marked as `internal` and `adhoc`. If you're agreeable, we could remove the next line
```
J9JDK_EXT_VERSION       := HEAD
```
and update the release instructions so that for the first milestone build we would change the definition to
```
J9JDK_EXT_VERSION       := $(VERSION_NUMBER_FOUR_POSITIONS)-m1
```
Whatever we decide, I think we should be consistent across all extension repos.

Reply:
I like the previous suggestion, but I think we should use `$(VERSION_NUMBER_FOUR_POSITIONS)-ea` rather than `m1`. `ea` can be used for all pre-releases and nightlies, it only needs to change for the release.

Reply:
That works for me: I'll change this (and the other extension repos) to say:
```
J9JDK_EXT_VERSION       := $(VERSION_NUMBER_FOUR_POSITIONS)-ea
```